pandas/tests/io/test_sql.py

```diff
@@ -1423,16 +1423,12 @@ def test_database_uri_string(self, test_frame1):
         tm.assert_frame_equal(test_frame1, test_frame3)
         tm.assert_frame_equal(test_frame1, test_frame4)
 
-        # using driver that will not be installed on Travis to trigger error
+    def test_pg8000_sqlalchemy_passthrough_error(self):
+        # using driver that will not be installed on CI to trigger error
         # in sqlalchemy.create_engine -> test passing of this error to user
-        try:
-            # the rest of this test depends on pg8000's being absent
-            import pg8000  # noqa:F401
-
-            pytest.skip("pg8000 is installed")
-        except ImportError:
-            pass
+        pytest.importorskip("pg8000")
```

Comment:
The current code imports `pg8000` and then skips the test if it's present, and continues if its **not** imported.

`importorskip` will import the module and continue **only if it is** imported. This behaviour seems to conflict with comments?

Reply:
Ah thanks for the correction. Used `@td.skip_if_installed` instead